### PR TITLE
Fix formatting of changelog

### PR DESCRIPTION
**Pull request by meshy**

### CODE REVIEW

**Comment by meshy** [comment]
@incuna/backend a quick one!

**Comment by adam-thomas** [comment]
Needs master merging, sorry (I just did a release).

**Comment by adam-thomas** [comment]
Other than that LGTM.  (Even if the `-` and `+` symbols added by GitHub's diff display make it rather confusing to review :P)

**Comment by meshy** [comment]
Nice one, ta! `:)`

**Comment by meshy** [comment]
any chance you could merge #29 before I merge master?

**Comment by adam-thomas** [comment]
Oh sorry, sure!  Although that one needs master merged in as well. :P

**Comment by meshy** [comment]
hahaha `:D`

**Comment by adam-thomas** [comment]
I would say "merged", but you did it :P

File: changelog.md [hunk 1]
```diff
@@ -1,83 +1,80 @@
...
+
+## Unreleased
 
 * Use html `buttons` instead of `inputs` for form submission.
 
-v1.1.0
-------
+## v1.1.0
 
 * Add the ability to reply to discussions by replying to notification emails.
```
Comment (adam-thomas) [inline changelog.md]: You do need to tweak these bullet points :)

Comment (meshy) [inline changelog.md]: well spotted! Ta!

**Comment by adam-thomas** [comment]
:+1:

**Comment by meshy** [comment]
Thanks `:)`

**Pull request merged**